go.mod

```diff
@@ -3,16 +3,21 @@ module github.com/flashbots/rpc-endpoint
 go 1.16
 
 require (
-	github.com/alicebob/gopher-json v0.0.0-20200520072559-a9ecdc9d1d3a // indirect
 	github.com/alicebob/miniredis v2.5.0+incompatible
 	github.com/ethereum/go-ethereum v1.10.13
 	github.com/go-redis/redis/v8 v8.11.4
-	github.com/gomodule/redigo v1.8.5 // indirect
 	github.com/google/uuid v1.3.0
+	github.com/jmoiron/sqlx v1.3.4
 	github.com/metachris/flashbotsrpc v0.4.0-alpha3
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.7.0
-	github.com/yuin/gopher-lua v0.0.0-20200816102855-ee81675732da // indirect
-	golang.org/x/crypto v0.0.0-20211117183948-ae814b36b871 // indirect
-	golang.org/x/sys v0.0.0-20211124211545-fe61309f8881 // indirect
 )
+
+require (
+	github.com/alicebob/gopher-json v0.0.0-20200520072559-a9ecdc9d1d3a // indirect
+	github.com/gomodule/redigo v1.8.5 // indirect
+	github.com/lib/pq v1.10.4 // indirect
+	github.com/yuin/gopher-lua v0.0.0-20200816102855-ee81675732da // indirect
+    golang.org/x/crypto v0.0.0-20211117183948-ae814b36b871 // indirect
+    golang.org/x/sys v0.0.0-20211124211545-fe61309f8881 // indirect
+)
```

Comment:
seems this file is a bit unclean, please re-run `go mod tidy`

Reply:
done